app/packs/src/decidim/vocdoni/admin/utils/create_vocdoni_election.js

```diff
@@ -136,6 +136,9 @@ export default class CreateVocdoniElection {
       electionType: {
         interruptible: electionMetadata.interruptible,
         secretUntilTheEnd: electionMetadata.secretUntilTheEnd
+      },
+      voteType: {
+        maxVoteOverwrites: -1
```

Comment:
is this value coming from the backend or for the moment it is always `-1`?

Reply:
Is always -1, as it's what I found for setting up any number of votes overwrites.

If this isn't set, then the maximum vote overwrites is 1 (this is something that has changed with the update to v0.0.4-alpha). It's not something that's inline with how we handle votes normally in Decidim.

As -1 is a magic number, and it isn't documented, I added a comment and extracted it to a constant variable in 3f8361b.